### PR TITLE
Add Retired Rank

### PR DESCRIPTION
**Pull request by DeathlyPlays**

### CODE REVIEW

**Comment by DeathlyPlays** [comment]
To avoid the branch fixing on #293 and all that since I did this on mobile I figured why not just do this over

File: config/config-example.js [hunk 1]
```diff
@@ -400,6 +400,12 @@ exports.grouplist = [
 		minigame: true,
 	},
 	{
+		symbol: '^',
+		id: "retired",
+		name: "Retired",
```
Comment (ghost) [inline config/config-example.js]: This means that on the pop up it'll say Global Retired. I feel like `Retired Auth` Would look a LOT nicer. The id can stay the same though

Comment (ghost) [reply config/config-example.js]: Also if you don't want Room retired. Make sure you add `globalonly: true,`

Comment (DeathlyPlays) [reply config/config-example.js]: [Development room discussion]

**Comment by DeathlyPlays** [comment]
@HoeenCoder can we add this now or close it etc

**Comment by ghost** [comment]
We're good on this, it's just there's a bug with custom symbols rn so I guess we are gonna wait (if I remember correctly)

**Comment by DeathlyPlays** [comment]
@HoeenCoder boop

**Comment by HoeenCoder** [comment]
On hold due to something upper staff related

**Comment by DeathlyPlays** [comment]
Oh, fair enough @HoeenCoder

**Comment by ghost** [comment]
??? @DeathlyPlays @HoeenCoder

**Comment by DeathlyPlays** [comment]
Staff decision that perhaps we should postpone this

**Pull request closed without merging**